app/javascript/react/utils/mapParamsHandler.ts

```diff
@@ -223,6 +224,38 @@ export const useMapParams = () => {
     debouncedUpdateURL(queryParams);
   }, [thresholdValues]);
 
+  const backToPreviousUserSettingsWithSessionAndStreamIdsReset =
```

Comment:
What say you?
`revertSettingsAndResetIds`
`revertUserSettingsAndResetIds`
`restoreSettingsAndClearIds`
`resetSessionStreamIdsAndRevertSettings`
`revertSettingsAndClearSessionStreamIds`

Reply:
`revertUserSettingsAndResetIds` 🤝 Thx!